r/R/utilities_expression.R

```diff
@@ -100,8 +107,7 @@ completeExpression <- function(expression, cell_ids) {
     return(expression)
   }
 
-  expression[, cell_ids := cell_ids]
```

Comment:
did you encounter any problems with in-place assignment here? I'm asking because I ran into some issues with the expression code a while ago.

Reply:
Yes, there was some side effects outside of the function (something in the parent function was being updated unintentionally). 

This kind of operation in incredibly fast in base R -- no way that it limits the speed of this function at all.